sources/packages/backend/apps/api/src/route-controllers/overaward/overaward.students.controller.ts

```diff
@@ -0,0 +1,60 @@
+import { Controller, Get } from "@nestjs/common";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import {
+  AllowAuthorizedParty,
+  RequiresStudentAccount,
+  UserToken,
+} from "../../auth/decorators";
+import { ClientTypeBaseRoute } from "../../types";
+import BaseController from "../BaseController";
+import {
+  OverawardBalanceAPIOutDTO,
+  StudentsOverawardAPIOutDTO,
+} from "./models/overaward.dto";
+import { OverawardControllerService } from "..";
+import { StudentUserToken } from "../../auth";
+
+@AllowAuthorizedParty(AuthorizedParties.student)
+@RequiresStudentAccount()
+@Controller("overaward")
+@ApiTags(`${ClientTypeBaseRoute.Student}-overaward`)
+export class OverawardStudentsController extends BaseController {
+  constructor(
+    private readonly overawardControllerService: OverawardControllerService,
+  ) {
+    super();
+  }
+
+  /**
+   * Get the overaward balance of a student.
+   * @returns overaward balance for student.
+   */
+  @ApiNotFoundResponse({
```

Comment:
I believe the ApiNotFoundResponse is not relevant to student API. Sorry missed this in previous review.